patches/minecraft/net/minecraft/client/server/LanServerDetection.java.patch

```diff
@@ -18,7 +18,7 @@
 +               final String ip;
 +
 +               // compress to short form if enabled in config
-+               if (net.minecraftforge.common.ForgeConfig.CLIENT.compressLanIPv6Addresses.get())
++               if (net.minecraftforge.common.ForgeConfig.COMMON.compressIPv6Addresses.get())
```

Comment:
Unless we are allowing breaking changes, I would suggest that this either respects both config options or that the PR waits until 1.20 releases.

Reply:
We're technically within the breaking changes window and the default of true still applies, but I've made a commit that keeps the old option working too if preferred. I'll leave it up to the core team to decide which behaviour they want through cherry-picking the commits.